python/lsst/ts/criopy/m1m3/acceleration_transformer.py

```diff
@@ -0,0 +1,107 @@
+# This file is part of M1M3 SS GUI.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org). See the COPYRIGHT file at the top - level directory
+# of this distribution for details of code ownership.
+#
+# This program is free software : you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["AccelerationTransformer"]
+
+import pathlib
+from typing import Any
+
+import pandas as pd
+import yaml
+
+
+class AccelerationTransformer:
+    """
+    Reads M1M3 configuration. Transform raw values into calibrated values, and
+    X Y Z accelerations.
+
+    Paramaters
+    ==========
+    config_dir : `None | str | pathlib.Path = None`
+        Configuration directory.
+    """
+
+    class Accelerometer:
+        def __init__(self, config: Any):
+            self.bias = config["Bias"]
+            self.sensitivity = config["Sensitivity"]
+            self.offset = config["Offset"]
+            self.scalar = config["Scalar"]
+
+        def __matmul__(self, value: Any) -> Any:
+            return (value - self.bias) * self.sensitivity * self.scalar - self.offset
+
+    def __init__(self, config_dir: None | str | pathlib.Path = None):
+        self.accelerometeres: list[Any] = []
+        self.distances: list[float] = [0, 0, 0]
+
+        if config_dir is not None:
+            self.load_config(config_dir)
+
+    def load_config(self, config_dir: str | pathlib.Path) -> None:
+        """Load M1M3 configuration files.
+
+        Parameters
+        ----------
+        config_dir : `str | pathlib.Path`
+            Directory where to look for configuration files.
+        """
+        config_dir = pathlib.Path(config_dir)
+        config_file = config_dir / "_init.yaml"
+
+        with open(config_file, "r") as file:
+            config = yaml.safe_load(file)
+
+        ac = config["AccelerometerSettings"]
+
+        self.accelerometeres = []
+
+        for accelerometer in range(8):
```

Comment:
8 accelerometers in total or only use 8 of them in this calculation? Please help to explain in the code comment.

Reply:
Introduced N_ACCELEROMETERS constant